poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
Are these changes expected? Should we revert changes here?

Reply:
this is to update idna to fix reported vulnerabilities